dispatcher/backend/src/utils/offliners.py

```diff
@@ -8,22 +8,25 @@
 # from common.constants import DISALLOW_CAPABILITIES
 from typing import List
 
+from common import constants
 from common.enum import Offliner
 
-od = collections.namedtuple("OfflinerDef", ["cmd", "std_output", "std_stats"])
+od = collections.namedtuple(
+    "OfflinerDef", ["cmd", "std_output", "std_stats", "publisher_flag"]
+)
 OFFLINER_DEFS = {
-    Offliner.freecodecamp: od("fcc2zim", True, False),
-    Offliner.gutenberg: od("gutenberg2zim", False, False),
-    Offliner.sotoki: od("sotoki", True, True),
-    Offliner.wikihow: od("wikihow2zim", True, True),
-    Offliner.ifixit: od("ifixit2zim", True, True),
-    Offliner.mwoffliner: od("mwoffliner", "outputDirectory", False),
-    Offliner.youtube: od("youtube2zim-playlists", True, False),
-    Offliner.ted: od("ted2zim-multi", True, False),
-    Offliner.openedx: od("openedx2zim", True, False),
-    Offliner.nautilus: od("nautiluszim", True, False),
-    Offliner.zimit: od("zimit", True, "statsFilename"),
-    Offliner.kolibri: od("kolibri2zim", True, False),
+    Offliner.freecodecamp: od("fcc2zim", True, False, True),
+    Offliner.gutenberg: od("gutenberg2zim", False, False, False),
```

Comment:
I guess that's because gutenberg has no publisher flag and this will change once it is released

Reply:
Yes